docs/curl/build.md

```diff
@@ -64,6 +64,45 @@ The output of the above command would be something similar to the following
 
 Congratulations! We have started a node successfully.
 
+## Build and start Blockchain DB node inside Docker
+### Prerequisites
+
+To build an docker image, the following are the prerequisites which should be installed.
+
+  - **Docker**: To generate cryptographic materials and build BCDB image.
+  - **Git**: To clone the code repository.
+  - **cURL**: To execute queries and transactions provided in the tutorial.
+
+### Build
+
+Build process includes two steps - crypto materials generation and docker image build
+
+#### Generate cryptographic materials 
+Minimal set of crypto materials includes 4 certificates - two for users - `admin` and `user`, one for server node - `node` and one for certificate authority - `CA` 
+To create minimal set of cryptographic materials, you can run:
+```
+./scripts/cryptoGen.sh sampleconfig
```

Comment:
mention what is `sampleconfig`? is it just a folder? should the folder exist already or will be created by the script itself?